examples.go

```diff
@@ -43,6 +43,6 @@ func printExamples(buf *bytes.Buffer, cmd *cobra.Command) {
 		}
 		buf.WriteString(`.. code-block::
 `)
-		buf.WriteString(fmt.Sprintf("\n   %s%s\n", comment, indentString(example, identChar)))
+		buf.WriteString(fmt.Sprintf("\n  %s%s\n", comment, indentString(example, identChar)))
```

Comment:
does this work for single examples and multi examples? the error I saw in your PR is the spacing issue only affecting single examples or examples with no `#`, I'll approve but consider this is why some tests here are important and we currently only test a particular example type instead of the 3 possible, no comment line, comment line single example, multi example